Assets/HoloToolkit/Input/Scripts/Utilities/Managers/MixedRealityCameraManager.cs

```diff
@@ -60,6 +60,10 @@ public enum DisplayType
         /// </summary>
         public event DisplayEventHandler OnDisplayDetected;
 
+        protected override void InitializeInternal()
```

Comment:
Perhaps we use these core scripts to demonstrate the auto resolve.
For example: For MixedRealityCameraManager.cs we must have CameraCache singleton resolved first.

Reply:
CameraCache isn't a singleton method.

Reply:
I still don't see a single place where this is getting used in the core toolkit. What about implementing this function in FocusManager.cs?

Reply:
I added it to Assets/HoloToolkit/Sharing/Scripts/SharingStage.cs so that it is ensured to be initialized when Assets/HoloToolkit-Examples/Sharing/SharingService/Scripts/RemoteHeadManager.cs  calls it.  Would you like it in FocusManager as well?